app/uk/gov/hmrc/fileupload/s3/S3JavaSdkService.scala

```diff
@@ -112,6 +118,18 @@ class S3JavaSdkService extends S3Service {
         s3Client.getObject(new GetObjectRequest(bucketName, key, versionId)).getObjectContent
       }
 
+
+  override def retrieveFileFromQuarantine(key: String, versionId: String)(implicit ec: ExecutionContext) = {
```

Comment:
getting a file from S3 is blocking, we need to ensure that we are using a thread pool, in this case we end up using `play.api.libs.concurrent.Execution.Implicits._` and we shouldn't